### PR TITLE
TRUNK-4634 "Dodgy - Write to static field from instance method" - extrafix

### PR DESCRIPTION
**Pull request by tomaszmueller**

### CODE REVIEW

File: api/src/main/java/org/openmrs/api/impl/ConceptServiceImpl.java [hunk 1]
```diff
@@ -1423,6 +1423,15 @@ public Concept getUnknownConcept() {
 	}
 	
 	/**
+	 * Sets unknownConcept using static method
+	 *
+	 * @param currentUnknownConcept
+	 */
+	public static void setStaticUnknownConcept(Concept currentUnknownConcept) {
```
Comment (rkorytkowski) [inline api/src/main/java/org/openmrs/api/impl/ConceptServiceImpl.java]: It should be private.

File: api/src/main/java/org/openmrs/api/impl/ObsServiceImpl.java [hunk 2]
```diff
@@ -865,6 +865,15 @@ public void setHandlers(Map<String, ComplexObsHandler> newHandlers) throws APIEx
 	}
 	
 	/**
+	 * Sets handlers using static method
+	 *
+	 * @param currentHandlers
+	 */
+	public static void setStaticHandlers(Map<String, ComplexObsHandler> currentHandlers) {
```
Comment (rkorytkowski) [inline api/src/main/java/org/openmrs/api/impl/ObsServiceImpl.java]: It should be private.

File: api/src/main/java/org/openmrs/api/impl/PatientServiceImpl.java [hunk 3]
```diff
@@ -1497,6 +1497,16 @@ public void setIdentifierValidators(Map<Class<? extends IdentifierValidator>, Id
 	}
 	
 	/**
+	 * Sets identifierValidators using static method
+	 *
+	 * @param currentIdentifierValidators
+	 */
+	public static void setStaticIdentifierValidators(
```
Comment (rkorytkowski) [inline api/src/main/java/org/openmrs/api/impl/PatientServiceImpl.java]: It should be private.

**Comment by tomaszmueller** [comment]
fixed and merged into one commit

**Comment by rkorytkowski** [comment]
Merged in a12f6a443bb6b750d49a4ae9304475bc26a9a037

**Pull request closed without merging**